src/server/apps/worldserver/worldserver.conf.dist

```diff
@@ -626,37 +626,37 @@ RealmZone = 1
 
 World.RealmAvailability = 1
 
-#
-#    StrictPlayerNames
-#        Description: Limit player name to language specific symbol set. Prevents character
-#                     creation and forces rename request if not allowed symbols are used
-#        Default:     0 - (Disable, Limited server timezone dependent client check)
-#                     1 - (Enabled, Strictly basic Latin characters)
-#                     2 - (Enabled, Strictly realm zone specific, See RealmZone setting,
-#                         Note: Client needs to have the appropriate fonts installed which support
-#                         the charset. For non-official localization, custom fonts need to be
-#                         placed in clientdir/Fonts.
-#                     3 - (Enabled, Basic Latin characters + server timezone specific)
-
-StrictPlayerNames = 0
-
 #
 #   StrictPlayerNames.Reserved
 #       Description: Use the Reserved Filter from DBC.
-#                    Prevents players from creating characters with profanity names.
+#                    Prevents Player, Pet & Charter names from containing reserved names.
```

Comment:
Is there much of a reason these are capitalized?

Reply:
I see them as names in this context, therefore capitalized